### DISCUSSION TITLE
New to all of this, how do I install it?

### DISCUSSION THREAD

**Discussion by ChloeKuiper** [webdjoe/pyvesync#225] [Q&A]
Hello, as the title states, I'm fairly new to the custom HASS stuff so I'm having a tough time figuring out how to install stuff like this.
I just got a Levoit LUH-D301S-WEU and I want to control it through my home assistant, but I can't use pip or brew or apt-get or anything to download stuff on the command line. I am running the Home Assistant OS on my Pi4 with 8GB RAM and 32GB SD-card.
![image](https://github.com/webdjoe/pyvesync/assets/117538511/0e79d4e7-1dba-4917-8dfd-ea07dd50c07c)

I have tried this integration but it does not work with the humidifier and I don't really know how to continue from this point onward.
![image](https://github.com/webdjoe/pyvesync/assets/117538511/5c808cb1-46aa-4e2c-ac89-43138fdaa070)

Thank you in advance,
Ole

**Comment by tobyl37** [comment]
I am pretty much at the same point.

**Comment by webdjoe** [comment]
You need to install via the integrations menu, it is already built into HomeAssistant: 
![image](https://github.com/webdjoe/pyvesync/assets/8164486/32b5503d-b5ad-4fb5-be86-cba6b0a43fb3)

**Reply by Tapion1985** [reply]
Hi, i have this issue as well...
problem is, with the "normal" vesync my fan is not shown

so i would like to try this one ;)

if i try to integrate it via HACS i get the following error:
"<Integration webdjoe/pyvesync> Repository structure for 2.1.12 is not compliant"

**Reply by miggsinablanket** [reply]
Hi, I am also looking to find out how to install this so I can utilize my Classic 300S humidifier in HAOS. I get the same error as above.

**Reply by cdnninja** [reply]
This github repo is a library.   You wouldn't install it into home assistant. You need the native integration via add integration.  That automatically uses this.

**Comment by webdjoe** [comment]
I don't know anything about the HACS integration. The Classic 300S should be supported by the native HA integration without installing from HACS
